README.md

```diff
@@ -105,7 +106,7 @@ I am running an acme-dns instance as a service for everyone wanting to get on in
 
 6) Edit config.cfg to suit your needs (see [configuration](#configuration))
 
-7) Run acme-dns. Please note that acme-dns needs to open a privileged port (53, domain), so it needs to be run with according privileges.
+7) Run acme-dns. Please note that acme-dns needs to open a privileged port (53, domain), so it needs to be run with correct privileges.
```

Comment:
Correct is too indefinitive too, maybe the correct (pun intended) word to use here would be "elevated"